src/main/java/com/google/api/generator/engine/writer/JavaWriterVisitor.java

```diff
@@ -120,4 +134,97 @@ public void visit(AssignmentExpr assignmentExpr) {
   private void space() {
     buffer.append(SPACE);
   }
+
+  /** =============================== STATEMENT =============================== */
+  public String visit(LineComment linecomment){
+      // Split comments by new line and add `//` to each line.
+      String[] sourceStrings = linecomment.comment().split("\\r?\\n");
+      for (int i = 0; i < sourceStrings.length; i++) {
+        sourceStrings[i] = "// " + sourceStrings[i];
+      }
+      String formattedSource =format(String.join("\n", sourceStrings));
```

Comment:
Curious if you've tried the following?

```
String.format("// %s", format(String.join("\n//", comment.comment().split("\\r?\\n")));
```

Reply:
cool, `format(String.format("// %s", String.join("\n//", comment.comment().split("\\r?\\n"))));` works for me!